### PR TITLE
Add gitlab_group_label resource

### PR DESCRIPTION
**Pull request by kirecek**
This MR adds a new resource `gitlab_group_label` to manage group labels.

### CODE REVIEW

File: gitlab/resource_gitlab_group_label.go [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+	for _, label := range labels {
+		if label.Name == labelName {
+			d.Set("description", label.Description)
+			d.Set("color", label.Color)
+			d.Set("name", label.Name)
+			found = true
+			break
+		}
+	}
+	if !found {
```
Comment (roidelapluie) [inline gitlab/resource_gitlab_group_label.go]: That should error, not be ignored

Comment (kirecek) [reply gitlab/resource_gitlab_group_label.go]: thx, added error in https://github.com/terraform-providers/terraform-provider-gitlab/pull/186/commits/9d000371b75eef256d7bcb244f450762e1f4020f

File: gitlab/resource_gitlab_group_label.go [hunk 2]
```diff
@@ -81,8 +83,7 @@ func resourceGitlabGroupLabelRead(d *schema.ResourceData, meta interface{}) erro
 		}
 	}
 	if !found {
-		log.Printf("[WARN] removing label %s from state because it no longer exists in gitlab", labelName)
-		d.SetId("")
+		return errors.New(fmt.Sprintf("label %s no longer exists in group %s", labelName, group))
```
Comment (roidelapluie) [inline gitlab/resource_gitlab_group_label.go]: I would say can not be found.
It does not exist or the user does not have the right to see it.

Comment (kirecek) [reply gitlab/resource_gitlab_group_label.go]: reworded the message

**Pull request merged**